### DISCUSSION TITLE
Interfacing with MSP430G2553

### DISCUSSION THREAD

**Discussion by girinath-ece** [olikraus/u8g2#1622] [Q&A]
Can i use this library for interfacing glcd ks0108(128x64) lcd display with msp430g2553 microcontroller.

**Comment by olikraus** [comment]
Yes, If you have a tool chain with link garbage collector, like gnu cc

**Reply by girinath-ece** [reply]
Thanks...will try and let u know

**Reply by girinath-ece** [reply]
I added the .h and .c files in the msp-gcc library directory. And the .c file compiled successfully when i imported u8g2.h but i am not able to call the functions. I dont know if i am doing it correctly, When i tried to do setup it is showing undefined reference error for functions. Can you help me with the setup. Since i am new to this can you help me with any example?

**Comment by olikraus** [comment]
Did you follow the instruction from here: https://github.com/olikraus/u8g2/wiki/Porting-to-new-MCU-platform ?

There are also many similar questions in the issues list.
